.travis.yml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: java
+
+git:
+  depth: 150
+
+jdk:
+  - oraclejdk7
+
+env: PATH=$PATH:$HOME/local/bin
+
+cache:
+  directories:
+  - $HOME/.m2
+  - $HOME/local
+
+notifications:
+  email:
+  - issues@reef.incubator.apache.org
+
+install: ./dev/travis-install-dependencies.sh
+
+script:
+  mvn clean install -ff -Dsurefire.useFile=false -DLOG_LEVEL=WARN
```

Comment:
One thing I dislike about the output of Travis build is the length of the log - my browser actually freezes from time to time trying to load it all or scroll it. It would be nice to reduce this to include only more-or-less meaningful messages.

What do you think about adding `--quiet` to mvn command line, so that at least Maven's [INFO]s about build order and excludes are not included? [INFO]s posted by our own tests will stay. I'm not sure whether this affects "downloading" messages.